frontend/src/features/projects/spl/tabs/SplTabs.tsx

```diff
@@ -76,7 +79,11 @@ export const SplTabs: React.FunctionComponent<ISplTabsProps> = ({
           title="Surplus Properties List"
           tabClassName={isTabInError(errors, SPPApprovalTabs.spl)}
         >
-          <SplTab isReadOnly={isReadOnly} setSubmitStatusCode={setSubmitStatusCode} />
+          <SplTab
+            isReadOnly={isReadOnly}
+            setSubmitStatusCode={setSubmitStatusCode}
+            goToGreTransferred={() => submitForm().then(() => goToGreTransferred())}
```

Comment:
Oh duh! The other day I was trying to use `handleSubmit` as a promise and forgot about `submitForm` 🤦

Reply:
we may need to enhance the error handling for this, but probably fine for now.